resume/index.css

```
@@ -0,0 +1,221 @@
+html {
+    font-family: Georgia;
+}
+
+body {
+    background-color: rgb(248, 239, 237);
+    margin: 0px;
+}
+
+.container {
+    position: relative;
+    text-align: center;
+}
+
+.rectangle {
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 200px 50vw 0 50vw;
+    border-color: rgb(234, 200, 190);
+}
```

This shouldn't use `border-width` to size itself; it should be sized like a regular container.

---

This I think is a good conversion, but we would have to move the content into the container.
```suggestion
.rectangle {
    width: 100%;
    height: 200px;
    background-color: rgb(234, 200, 190);
}
```